src/js/components/SharedComponents/ProgressComponent.jsx

```diff
@@ -45,14 +46,36 @@ export default class Progress extends React.Component {
 
         for (let i = 1; i <= this.props.totalSteps; i++) {
             if (i < this.props.currentStep) {
-                progressBar.push(<li key={i} className="usa-da-progress-bar-step-done"><a href={stepLink[i-1] +"/"+ this.props.id} ><span className="step">{i}</span></a></li>);
-                progressLabels.push(<li key={i} className="usa-da-progress-bar-step-done"><span className="name"><a href={stepLink[i-1] +"/"+ this.props.id} >{stepNames[i - 1]}</a></span></li>);
-            } else if (i === this.props.currentStep) {
-                progressBar.push(<li key={i} className="usa-da-progress-bar-step-current"><span className="step">{i}</span></li>);
-                progressLabels.push(<li key={i} className="usa-da-progress-bar-step-current"><span className="name">{stepNames[i - 1]}</span></li>);
-            } else {
-                progressBar.push(<li key={i} className="usa-da-progress-bar-step"><span className="step">{i}</span></li>);
-                progressLabels.push(<li key={i} className="usa-da-progress-bar-step"><span className="name">{stepNames[i - 1]}</span></li>);
+                progressBar.push(
```

Comment:
While we're already refactoring this, can we fix this so we don't repeat the same code multiple times? Something like:

    if () {
         className = something
    }
    else if () {
        className = somethingelse
    }
    progressBar.push(
        <li key={i} className=className>
            <span className="step"></span>
        </li>
    );

The difficult part would be deciding how to make the span a link, but that shouldn't be too difficult and it would clear up the code a whole lot.